### PR TITLE
dockutil: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/osx/dockutil.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Añade una aplicación al final del dock del usuario actual:
+
+`dockutil --add {{ruta/a/la/aplicación}}`
+
+- Sustituye una aplicación por otra en el dock del usuario actual:
+
+`dockutil --add {{ruta/a/la/aplicación}}} --replacing '{{etiqueta_de_elemento_del_dock}}'`
+
+- Añade un directorio con opciones de visualización y lo muestra como un icono de carpeta o pila:
```
Comment (tricantivu) [inline pages.es/osx/dockutil.md:14]:
```suggestion
- Añade un directorio con opciones de visualización y muestralo como un icono de carpeta o pila:
```

File: pages.es/osx/dockutil.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`dockutil --add {{vnc://ejemplo_servidor.local}} --label '{{VNC de ejemplo}}' --after {{etiqueta_de_elemento_del_dock}}`
+
+- Elimina una aplicación del dock dado su nombre de etiqueta en el dock:
+
+`dockutil --remove '{{etiqueta_de_elemento_del_dock}}'`
+
+- Añade un espaciador en una sección tras una aplicación:
+
+`dockutil --add '' --type {{spacer|small-spacer|flex-spacer}} --sección {{apps}} --after {{etiqueta_de_elemento_del_dock}}`
```
Comment (tricantivu) [inline pages.es/osx/dockutil.md:28]: I think you might have meant [`--section`](https://github.com/kcrawford/dockutil/blob/main/Sources/DockUtil/DockUtil.swift#L227)

```suggestion
`dockutil --add '' --type {{spacer|small-spacer|flex-spacer}} --section {{apps}} --after {{etiqueta_de_elemento_del_dock}}`
```

File: pages.es/osx/dockutil.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Sustituye una aplicación por otra en el dock del usuario actual:
+
+`dockutil --add {{ruta/a/la/aplicación}}} --replacing '{{etiqueta_de_elemento_del_dock}}'`
+
+- Añade un directorio con opciones de visualización y lo muestra como un icono de carpeta o pila:
+
+`dockutil --add {{/ruta/al/directorio}} --view {{grill|fan|list|auto}} --display {{folder|stack}}`
+
+- Añade la URL de un elemento del dock tras otro elemento:
```
Comment (tricantivu) [inline pages.es/osx/dockutil.md:18]:
```suggestion
- Añade la URL de un elemento del dock después de otro elemento:
```

File: pages.es/osx/dockutil.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`dockutil --add {{vnc://ejemplo_servidor.local}} --label '{{VNC de ejemplo}}' --after {{etiqueta_de_elemento_del_dock}}`
+
+- Elimina una aplicación del dock dado su nombre de etiqueta en el dock:
+
+`dockutil --remove '{{etiqueta_de_elemento_del_dock}}'`
+
+- Añade un espaciador en una sección tras una aplicación:
```
Comment (tricantivu) [inline pages.es/osx/dockutil.md:26]:
```suggestion
- Añade un espaciador en una sección después de una aplicación:
```

**Review by tricantivu — COMMENTED** [review]
Thanks for translating the page @kant. I have a few suggestions, let me know if you agree or not.

File: pages.es/osx/dockutil.md (hunk 1 above)
Comment (kant) [reply pages.es/osx/dockutil.md:14]: `muestralo` is not properly conjugated verb. The proper way is `lo muestra`, `lo visualiza`, `lo presenta`.

**Review by tricantivu — APPROVED** [review]

**Comment by kant** [comment]
> Outdated page when comparing with the English page

Updated

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**